lib/tasks/manually_counted_members.rake

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Schweizer Blasmusikverband. This file is part of
+#  hitobito_sbv and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sbv.
+
+namespace :manually_counted_members do
```

Comment:
I would expect to find this in a **group** namespace rather than creating it's own here.

how about: group:manual_member_count_activate ...